lib/caasp.pm

```diff
@@ -33,6 +33,11 @@ sub process_reboot {
     script_run("reboot", 0) if $trigger;
 
     reset_consoles;
+    if (check_var('VIRSH_VMM_FAMILY', 'xen')) {
+        sleep 7;
+        select_console 'sut';
+    }
```

Comment:
it does not look correct to me to have something like this in `lib/caasp.pm`. That is valid regardless of the product but depending on the backend, right?